index.html

```diff
@@ -38,6 +39,140 @@ <h2>I'am a Software Developer</h2>
             </ul>
         </div>
     </div>
+    <div class="grid-container">
+        <section class="myworks">
+            <h3>My Recent Works</h3>
+            <div class="firstcard">
+                <img src="imgs/placeholder.svg" alt="placeholder" class="placeholder">
+                <h2>Multi-Post Stories</h2>
```

Comment:
[OPTIONAL]

- [ ] You tried sticking to your chosen template and your work looks so great. However, this `<h2>Multi-Post Stories</h2>` does not have the same color and size as the one in the design. Please consider applying the correct color and size 👍.

your design:
<img width="158" alt="chrome_6o2NGjIcPh" src="https://user-images.githubusercontent.com/81468675/143661053-a4acec0e-4fc7-4fc7-956e-2a3025b5de74.png">

original design:
<img width="159" alt="chrome_xwWQK97lb9" src="https://user-images.githubusercontent.com/81468675/143661064-0e85dc8d-6db5-4bdb-8985-f0a52d676149.png">